packages/main/src/plugin/container-registry.ts

```diff
@@ -774,6 +774,21 @@ export class ContainerProviderRegistry {
     return crypto.createHash('sha512').update(imageName).digest('hex');
   }
 
+  async pingContainerEngine(providerContainerConnectionInfo: ProviderContainerConnectionInfo): Promise<unknown> {
+    let telemetryOptions = {};
+    try {
+      return this.getMatchingEngineFromConnection(providerContainerConnectionInfo).ping();
+    } catch (error) {
+      telemetryOptions = { error: error };
+      throw error;
+    } finally {
+      this.telemetryService
+
```

Comment:
Maybe remove this line

Reply:
@jeffmaury you don't want to track people checking with ping ?

Reply:
fixed